pycbc/events/coinc_rate.py

```diff
@@ -82,10 +82,38 @@ def combination_noise_coinc_rate(rates, slop):
         Value is expected coincidence rate in the combination, units Hz
     """
     # multiply product of trigger rates by the overlap time
-    allowed_area = multiifo_noise_coincident_area(list(rates), slop)
+    log_rates = {k: numpy.log(r) for (k, r) in rates.items()}
+    combo_coinc_rate = combination_noise_coinc_rate_log(log_rates, slop)
```

Comment:
don't you want to do numpy.exp on this? otherwise it will return the log of a rate.